README.md

```diff
@@ -8,8 +8,17 @@ access to filesystems, and network connectivity.
 
 Building
 ---
+There are a few methods for building the binary.
+_We use [go modules](https://github.com/golang/go/wiki/Modules), so you need to build with Go 1.11 or newer._
 
-We use [go modules](https://github.com/golang/go/wiki/Modules), so you need to build with Go 1.11 or newer. `go build` or `make` should be sufficient to generate a working firectl binary.
+#### Make
```

Comment:
I think we can remove the section heading, since I don't think we need a separate Make and Docker section.

Reply:
Can you remove the "#### Make" section heading?